static/js/filter.js

```diff
@@ -205,7 +212,7 @@ function encodeOperand(operand) {
 }
 
 function decodeOperand(encoded, operator) {
-    if (operator !== 'pm-with' && operator !== 'sender' && operator !== 'from') {
+    if (operator !== 'huddle-with' && operator !== 'pm-with' && operator !== 'sender' && operator !== 'from') {
```

Comment:
We should fix the existing code here in two different ways:
* Make the if statement check for the positive condition, not the negative condition, and return the result of robust_uri_decode in the `if` block, and vice versa for the `replace` call.
* We should say extract some sort of helper like `is_person_operator`, or use a more concise idiom like `_.contains['pm-with', ...], operator)`.